src/main/kotlin/camp/nextstep/edu/step/step2/domain/lotto/WinningLotto.kt

```diff
@@ -1,11 +1,16 @@
 package camp.nextstep.edu.step.step2.domain.lotto
 
+import camp.nextstep.edu.step.step2.domain.number.Number
+
 data class WinningLotto(
     val winningLotto: Lotto,
+    val bonusNumber: Number
 ) {
 
     init {
         require(winningLotto.numbers.isNotEmpty()) { "당첨번호가 입력되지 않았습니다." }
+        // winningLotto 내 Number들과 bonusNumber가 중복되는지 검사
+        require(!winningLotto.numbers.contains(bonusNumber)) { "보너스 번호는 당첨 번호와 중복될 수 없습니다." }
```

Comment:
```suggestion
        require(!winningLotto.contains(bonusNumber)) { "보너스 번호는 당첨 번호와 중복될 수 없습니다." }
```
메시지를 던지는 구조로 변경해보세요 😄

Reply:
한번 말씀해주신 구조로 변경해보겠습니다!!